win32/devdoc/threadpool_win32.md

```diff
@@ -56,6 +56,24 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_WIN32_01_025: [** `threadpool_create` shall obtain the PTP_POOL from the execution engine by calling `execution_engine_win32_get_threadpool`. **]**
 
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_011: [** Otherwise, `threadpool_create` shall switch the state to OPENING. **]**
+
+[Will be moved from threadpool_open.]
```

Comment:
>Will be moved from threadpool_open.] [](http://example.com/codeflow?start=1&length=36)

Same here, a simple note is enough. /consider

Reply:
Done